index.html

```diff
@@ -88,49 +96,224 @@
 <script type="text/javascript" src="https://npmcdn.com/simple-statistics@2.0.0/dist/simple-statistics.min.js"></script>
 <script>
   var data = [];
-  var options = {
+    var options = {
     yearRange: 'all'
   };
+  var svg, x, y, xAxis, yAxis, xh, yh; 
+  var margin = { top: 15, right: 15, bottom: 30, left: 45 };
+  var width = 640 - margin.right - margin.left;
+  var height = 400 - margin.top - margin.bottom;
+  var xpadding = 0;
+  var ypadding = 0;
+  
+  // Creating GRID function
+  function make_x_axis() {        
+    return d3.svg.axis()
+        .scale(x)
+         .orient("bottom")
+         .ticks(5)
+  }
 
-  var svg;
+  function make_y_axis() {        
+    return d3.svg.axis()
+        .scale(y)
+        .orient("left")
+        .ticks(5)
+  }
 
-  // FETCH DATA
 
+  // FETCH DATA
+  d3.json('data.json', function (error, json) {
+    if (error) { throw error; }
+    data = json;
+    setup();
+    update();
+  });
   function setup() {
     // SVG
-
     svg = d3.select('#chart')
-      .append('svg');
-
+      .append('svg')
+      .attr('width', width + margin.right + margin.left)
+      .attr('height', height + margin.top + margin.bottom)
+      .append('g')
+      .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
     // SCALES
-
+    x = d3.scale.linear()
+      .domain([0, d3.max(data, function (d) { return d.unemployment; })])
+      .range([0, width])
+      .nice();
+    y = d3.scale.linear()
+      .domain(d3.extent(data, function (d) { return d.employment; }))
+      .range([height, 0])
+      .nice();
+    
     // AXES
-
+    //Define X axis
+      xAxis = d3.svg.axis()
+                .scale(x)
+                .orient("bottom")
+                .ticks(5);
+    //Define Y axis
+      yAxis = d3.svg.axis()
+                .scale(y)
+                .orient("left")
+                .ticks(5);
     // EVENT HANDLERS
-
     d3.select('#year-range').on('change', function () {
       options.yearRange = d3.event.target.value;
       update();
     });
   }
-
   function update() {
     // TRANSFORM
     txData = data.slice();
-
     if (options.yearRange !== 'all') {
       var yearRange = options.yearRange.split('-');
       txData = txData.filter(function (d) {
         return d.year >= yearRange[0] && d.year <= yearRange[1];
       });
     }
-
     // SCATTERPLOT
+    var points = svg.selectAll('.point')
+      .data(data);
```

Comment:
Note that you're using the untransformed data. This is why the dropdown menu doesn't work. We need to use `txData` here, instead of `data`. Same on line 265.